lib/src/util/react_wrappers.dart

```diff
@@ -120,7 +117,7 @@ final bool _canUseExpandoOnReactElement = (() {
   return true;
 })();
 
-/// A cache of props for a given [ReactElement].
+/// A cache of [UiComponent.props] for a given [ReactElement].
```

Comment:
This is a little misleading, since not all `ReactElement` doesn't always have a associated `UiComponent` (only when it's mounted).